starboard/CHANGELOG.md

```diff
@@ -5,205 +5,224 @@ This file will be updated each time a new Starboard version is released.
 Each section in this file describes the changes made to the Starboard interface
 since the version previous to it.
 
-**NOTE: Starboard versions 9 and below are no longer supported.**
+**NOTE: Starboard versions 13 and older are no longer supported.**
 
 ## Version 16
+A key update in Starboard version 16 is the adoption of POSIX APIs.
+For a full overview of Starboard POSIX migrations, please refer to
+[migration guide](/starboard/doc/starboard_16_posix.md).
 
-## Deprecated `SbLogIsTty`
-Removed, will use `isatty(fd)` if needed.
+### New Features / Support
 
-### Added Socket Waiter API for POSIX.
-Introduced Starboard functions SbPosixSocketWaiterAdd, SbPosixSocketWaiterRemove
-and callback function SbPosixSocketWaiterCallback to support POSIX based socket
-APIs.
-
-## Deprecated `Accessibility` header
-
-Accessibility Starboard API has been deprecated, an extension in
-`starboard/extension/accessibility.h` is available instead. The removal
-includes the following functions:
-* `SbAccessibilityGetCaptionSettings`
-* `SbAccessibilityGetDisplaySettings`
-* `SbAccessibilityGetTextToSpeechSettings`
-* `SbAccessibilitySetCaptionsEnabled`
-
-## Added new configuration constant `kHasPartialAudioFramesSupport`
+#### Added new configuration constant `kHasPartialAudioFramesSupport`
 Set this to true if your platform supports partial audio frames.
 
-### Deprecated `SbMutex`, `SbConditionVariable` and `SbThread`.
-The standard POSIX `pthread` APIs replace the Starboard concurrency
-primitives.
-
-### Migrate the `SbThreadSampler` to use `pthread`.
-Switched the `SbThreadSampler` API to use `pthread` instead of `SbThread`.
-
-### Added support for pthread create attributes.
-The standard pthread APIs `pthread_attr_init`, `pthread_attr_destroy`,
-`pthread_attr_getdetachstate`, `pthread_attr_getstacksize`,
-`pthread_attr_setdetachstate`, `pthread_attr_setstacksize` were added.
-
-### Deprecated the `SbThreadSetName` and `SbThreadGetName` APIs.
-Replaced the `SbThreadSetName`/`SbThreadGetName` with the POSIX
-`pthread_setname_np`/`pthread_getname_np`.
-
-### Deprecated the `SbThreadLocalKey` APIs.
-Replaced the `SbThreadLocalKey` with the POSIX `pthread_key_t`.
-
-### Deprecated `SbOnce`
-Replaced the `SbOnce` with the POSIX `pthread_once`.
-
-### Deprecated `SbThreadSleep`
-Replaced the `SbThreadSleep` with the POSIX usleep() defined in the
-`<unistd.h>` header.
-
-### MAP_EXECUTABLE_MEMORY changed from build-time to runtime config
-SB_CAN_MAP_EXECUTABLE_MEMORY has been refactored into a run-time configuration
-constant `kSbCanMapExecutableMemory`.
-
-### Deprecated `SbThreadYield`
-Replaced the `SbThreadYield` with the POSIX sched_yield() defined in the
-`<sched.h>` header.
-
-### x86 ABI removed for Evergreen
+#### Added support of POSIX APIs
```

Comment:
A lot of these apis are still replacements for Starboard APIs, the migration just isn't complete yet (i.e. much of SbFile and SbSocket still needs to be migrated, but it's targeted for SB16). With that in mind, it might end up making sense to move some of these into the "Migrate" section—but that could also wait until it actually happens.

Reply:
Note i.e. that `SbFileOpen` is already mapped to `open` in the migration guide: https://github.com/youtube/cobalt/blob/main/starboard/doc/starboard_16_posix.md?plain=1#L189

Reply:
Hi @andrewsavage1 , I agree with you that some of the APIs in `New Features / Support` are replacement of existing SB APIs. I will move them to migration section.

Reply:
Hi @andrewsavage1 , since those APIs are added to SB16 but the old functions are not actually removed. Do you think it is okay to leave it at `New Features` and move it to `Updates` once those functions are removed?

Reply:
Leaving as is for now in the interests of having a first pass in, we can follow up.